src/components/YourTripsPage/YourTripsPage.js

```diff
@@ -1,16 +1,18 @@
 import styled from 'styled-components'
 import TripCard from '../TripCard/TripCard'
 
-export default function YourTripsPage({ cards, setCards }) {
+export default function YourTripsPage({ tripCards, setTripCards }) {
```

Comment:
Maybe just `TripPage`? That way the name is more generic? But that's just my opinion :)